src/com_tjvendors/admin/sql/install.mysql.utf8.sql

```diff
@@ -1,33 +1,33 @@
 CREATE TABLE IF NOT EXISTS `#__tjvendors_passbook` (
 `id` INT(11) UNSIGNED NOT NULL  AUTO_INCREMENT,
-`vendor_id` INT(11)  NOT NULL ,
-`currency` VARCHAR(255)  NOT NULL ,
+`vendor_id` INT(11)  NOT NULL DEFAULT 0,
+`currency` VARCHAR(255)  NOT NULL DEFAULT '',
 `total` DOUBLE(15,2)  NOT NULL ,
 `credit` DOUBLE(15,2)  NOT NULL ,
 `debit` DOUBLE(15,2)  NOT NULL ,
-`reference_order_id` VARCHAR(255)  NOT NULL ,
+`reference_order_id` VARCHAR(255)  NOT NULL DEFAULT '',
```

Comment:
can we not mark column as NULLable instead of setting default as blank?

Reply:
Joomla itself has used same for column for varchar data type